db/data_migration/20230515143400_remove_central_african_republic_embassy_special_case.rb

```diff
@@ -0,0 +1,14 @@
+location = WorldLocation.find_by!(slug: "central-african-republic")
+organisation = WorldwideOrganisation.find_by!(slug: "british-embassy-kinshasa")
+organisation.world_locations += [location] unless organisation.world_locations.include?(location)
+
+location.reload
+
+embassy = Embassy.new(location)
```

Comment:
I think I'm missing something here, what does this line do (i.e. obviously it creates an embassy, but Embassies aren't stored in the database anywhere so I was reading this as having no effect)

Reply:
We use the [`Embassy#organisations_with_embassy_offices`](https://github.com/alphagov/whitehall/blob/d5e8993a2059c0deddb98438f9e85988ec151938/app/models/embassy.rb#L8) method to detect whether the embassy organisations for that location match what we expect them to be after we've changed the data. Does that make sense?

Reply:
Oh 🤦‍♀️ 🤦‍♀️ 🤦‍♀️  sorry yes was thinking about that failing fast stuff separately from this